provisioning/service/samples/run_bulk_operation.js

```diff
@@ -5,7 +5,18 @@
 
 var provisioningServiceClient = require('azure-iot-provisioning-service').ProvisioningServiceClient;
 
-var serviceClient = provisioningServiceClient.fromConnectionString(process.argv[2]);
+var argv = require('yargs')
+  .usage('Usage: $0 --connectionString <DEVICE PROVISIONING CONNECTION STRING> ')
+  .option('connectionstring', {
```

Comment:
>s [](start = 21, length = 1)

typo here (I think the S should be uppercase)

Reply:
fixed it